capstone/src/main/webapp/flags_script.js

```diff
@@ -20,36 +20,32 @@ const MIN_MAP_ZOOM = 12;
 const DEFAULT_MAP_ZOOM = 15;
 const MAX_MAP_ZOOM = 18;
 const gradient = [
-  "rgba(0, 255, 255, 0)",
-  "rgba(0, 255, 255, 1)",
-  "rgba(0, 191, 255, 1)",
-  "rgba(0, 127, 255, 1)",
-  "rgba(0, 63, 255, 1)",
-  "rgba(0, 0, 255, 1)",
-  "rgba(0, 0, 223, 1)",
-  "rgba(0, 0, 191, 1)",
-  "rgba(0, 0, 159, 1)",
-  "rgba(0, 0, 127, 1)",
-  "rgba(63, 0, 91, 1)",
-  "rgba(127, 0, 63, 1)",
-  "rgba(191, 0, 31, 1)",
-  "rgba(255, 0, 0, 1)"
+  'rgba(0, 255, 255, 0)',
+  'rgba(0, 255, 255, 1)',
+  'rgba(0, 191, 255, 1)',
+  'rgba(0, 127, 255, 1)',
+  'rgba(0, 63, 255, 1)',
+  'rgba(0, 0, 255, 1)',
+  'rgba(0, 0, 223, 1)',
+  'rgba(0, 0, 191, 1)',
+  'rgba(0, 0, 159, 1)',
+  'rgba(0, 0, 127, 1)',
+  'rgba(63, 0, 91, 1)',
+  'rgba(127, 0, 63, 1)',
+  'rgba(191, 0, 31, 1)',
+  'rgba(255, 0, 0, 1)'
 ];
  
 /* geolocation api */
 function getUserLocation(){
   let time = new Date();
   if (time.getHours() >= SWITCH_HOUR) {
     document.body.style.backgroundColor = '#614051';
-    document.getElementById('logo').src = 'images/logo.png';
-    document.getElementById("top_nav").className = "navbar navbar-expand-md navbar-dark bg-dark sticky-top";
-    document.getElementById("pac").style.color = "white";
-    document.getElementById("pac").className = "bg-dark";  
   }
   $.ajax({
     type : 'POST',
     data: '', 
-    url: "https://www.googleapis.com/geolocation/v1/geolocate?key=AIzaSyCgozira2dGlwMHT_WgQpmg84fk3VhRglM", 
+    url: 'https://www.googleapis.com/geolocation/v1/geolocate?key=AIzaSyCgozira2dGlwMHT_WgQpmg84fk3VhRglM', 
```

Comment:
Is this an API key? We should be storing API keys in a .gitignore file to keep them secure (and not publicly viewable on github). Can you move this one and any others in the codebase?

Reply:
Issue has been resolved in the "move keys" commit